### PR TITLE
chore(functions): Revise test coverage for CI utility updates

### PR DESCRIPTION
**Pull request by grayside**
* Rename $name in functions tests to $entryPoint
* Fix env_vars sample DeployTest and SystemTest to use first test case env var.

### CODE REVIEW

File: functions/env_vars/test/DeployTest.php [hunk 1]
```diff
@@ -37,7 +37,19 @@ class DeployTest extends TestCase
...
+    /**
+     * Deploy the Function.
+     *
+     * Overrides CloudFunctionLocalTestTrait::doDeploy().
+     */
+    private static function doDeploy()
+    {
+        $cases = self::cases();
+        $case = reset($cases);
+        self::$fn->deploy([$case['varName'] => $case['varValue']]);
```
Comment (bshaffer) [inline functions/env_vars/test/DeployTest.php:51]: I am confused as to what's happening here

File: functions/helloworld_http/composer.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
     "require": {
         "google/cloud-functions-framework": "^0.6"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.4"
```
Comment (bshaffer) [inline functions/helloworld_http/composer.json:6]: We should use the phpunit in `../../testing/vendor/bin/phpunit`. Do you have a reason you're adding this here?

Comment (grayside) [reply functions/helloworld_http/composer.json:6]: Accident, will revert.

File: functions/env_vars/test/DeployTest.php (hunk 1 above)
Comment (grayside) [reply functions/env_vars/test/DeployTest.php]: This has a bug, which I've fixed. Also clarified the code a bit and added a comment.

tl;dr: The env_var sample got merged but the DeployTest and SystemTest had to have been failing without a custom variable in the running environment. I've switched the code to pull that var from the dataProvider, which only works because there's just one test case.

Fixing this more completely is out of scope, I only did this to make sure this PR could pass.

File: functions/env_vars/test/DeployTest.php [hunk 3]
```diff
@@ -37,7 +37,20 @@ class DeployTest extends TestCase
     use CloudFunctionDeploymentTrait;
     use TestCasesTrait;
 
-    private static $name = 'envVar';
+    private static $entryPoint = 'envVar';
+
+    /**
+     * Deploy the Function.
+     *
+     * Overrides CloudFunctionLocalTestTrait::doDeploy().
```
Comment (ace-n) [inline functions/env_vars/test/DeployTest.php:45]: Nit: this seems like `CloudFunctionDeploymentTrait`?

File: functions/helloworld_http/composer.json [hunk 4]
```diff
@@ -1,5 +1,7 @@
 {
     "require": {
         "google/cloud-functions-framework": "^0.6"
+    },
+    "require-dev": {
```
Comment (bshaffer) [inline functions/helloworld_http/composer.json]: remove unneeded `require-dev`!

**Review by bshaffer — APPROVED** [review]
one nit, otherwise looks good!
